### ISSUE TITLE
Dark mode for docs?

### ISSUE THREAD

**Issue by alirezaalavi87** [Keats/gutenberg#2390]
Hi. thanks for the very cool project.

I noticed the [docs](https://www.getzola.org/documentation) doesn't have a dark mode and I feel like I'm going blind trying to read the docs. specially for an SSG I think not having good looking docs is a bad look.

the darkreader plugin also doesn't work on it for some reason. would be really great if dark mode was added to the website.

**Comment by Keats** [comment]
It's planned when the docs will be changed/rewritten but that's very far from the top of the list

**Comment by charlesrocket** [comment]
@alirezaalavi87 I use the browser's reader with such websites, though the code formatting is less distinguishable from time to time.

**Comment by alirezaalavi87** [comment]
@charlesrocket yep darkreader is working now. at the time of this issue, darkreader wasn't working for some reason
